mapeditor/validator.h

```diff
@@ -30,13 +31,18 @@ class Validator : public QDialog
 		bool critical;
 		
 		Issue(const QString & m, bool c): message(m), critical(c) {}
+
+		bool operator <(const Issue & other) const
+		{
+			return message < other.message;
```

Comment:
technically you should also take the second field into account

Reply:
Thought it doesn't matter for this struct. Will fix.

Reply:
> Thought it doesn't matter for this struct. Will fix.

I think it doesn't indeed - guess you don't have same messages that can be true and false